go-samson-secret-puller/secrets.go

```diff
@@ -0,0 +1,36 @@
+package samsonsecretpuller
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+// Secrets returns a map of secret names and values
+func Secrets() (map[string]string, error) {
+	s, err := readSecrets("/secrets")
+	return s, err
+}
+
+func readSecrets(d string) (map[string]string, error) {
+	s := make(map[string]string)
+	files, err := ioutil.ReadDir(d)
```

Comment:
As far as I can tell from [the Ruby implementation](https://github.com/zendesk/samson_secret_puller/blob/62853607f095b1d4bdd18ab4442b07dbfd4a873d/gem/lib/samson_secret_puller.rb#L50), it looks like it doesn't support nested folders?